src/main/java/com/google/cloud/pontem/CloudSpannerDatabaseBackup.java

```diff
@@ -396,8 +403,22 @@ public static void main(String[] args) throws Exception {
             .withInstanceId(options.getInputSpannerInstanceId())
             .withDatabaseId(options.getInputSpannerDatabaseId());
 
-    // STEP 2a: Save DDL to disk
+    // STEP 2a: Check proposed backup location
     final Util util = new Util();
+    if (!options.getWhetherToOverwriteGcsFileBackup()
+        && (util.getNumGcsBlobsInGcsFilePath(
+                options.getProjectId(),
+                Util.getGcsBucketNameFromDatabaseBackupLocation(options.getOutputFolder()),
+                Util.getGcsFolderPathFromDatabaseBackupLocation(options.getOutputFolder())))
```

Comment:
Nit: Could you extract the blob size into a variable to make the if statement more readable?